internal/executor/service/job_manager_test.go

```diff
@@ -10,15 +10,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 
-	"github.com/G-Research/armada/internal/common/util"
+	commonUtil "github.com/G-Research/armada/internal/common/util"
 	podchecksConfig "github.com/G-Research/armada/internal/executor/configuration/podchecks"
 	"github.com/G-Research/armada/internal/executor/context"
+	context2 "github.com/G-Research/armada/internal/executor/context/fake"
```

Comment:
Is `context2` appropriate here? Would it be better to call it `fakecluster` or `fakecontext`?